content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the Realtime Client Library SDK."
+section: realtime
```

Comment:
Invalid metadata. See Miro for a list of supported metadata for the new toolchain.

Reply:
Done